### DISCUSSION TITLE
AVX2 is a  minimum requirement？

### DISCUSSION THREAD

**Discussion by Ecthellin203** [ggml-org/llama.cpp#7723] [Q&A]
I got a 7900XTX with E5v2 CPUs. When Ollama runs models, it will cause an error“Error: llama runner process has terminated: exit status 0xc000001d”. 
I want to confirm if this is due to the lack of AVX2?

**Comment by metal3d** [comment]
I'm working on an "old" computer (13 years old) with a RTX 3070 and 24Go RAM. I don't have AVX2 support on this CPU, only AVX. And it works.

Linux Fedora 40.

**Reply by Ecthellin203** [reply]
Obviously, Nvidia does much better than AMD in this regard. Now i believe this is a problem with ROCm, not llama.cpp.
